src/ast/data_type.rs

```diff
@@ -196,6 +206,8 @@ impl fmt::Display for DataType {
             DataType::Dec(info) => {
                 write!(f, "DEC{info}")
             }
+            DataType::BigNumeric => write!(f, "BIGNUMERIC"),
+            DataType::BigDecimal => write!(f, "BIGDECIMAL"),
```

Comment:
You're not adding the precision/scale for those types. Any reason why?

Are you only focused on literals?

Reply:
I just considered for literals at first, but found that `BigNumeric` & `Numeric` with precision/scale are used in bq scripting. https://cloud.google.com/bigquery/docs/reference/standard-sql/data-types#parameterized_data_types

I'll implement because this is needed in the future.